addons/mail/static/src/js/onchange_on_keydown.js

```diff
@@ -0,0 +1,65 @@
+/** @odoo-module **/
+
+import { patch } from "@web/core/utils/patch";
```

Comment:
can we add a small test to check that the onchange value is correct ? (e.g. if the input is empty and you press "e", the event.value should be "e", not the "previous value" of the input)